XSLT/nashvillep15769coll19qdctomods.xsl

```diff
@@ -0,0 +1,332 @@
+      <dc:language>spa</dc:language>
+      or
+      <dc:language>English; German</dc:language>
+      <dc:language>eng; deu</dc:language>
+    The list of languages (pLang, above) is exhaustive for the current set of
+    records. The following template matches the first dc:language element, tokenizes
+    the value(s), and then does some comparisons vs the values in pLang.
+
+  -->
+  <xsl:template match="dc:language[1]">
+    <xsl:variable name="lang-tokens" select="tokenize(., ';')"/>
+    <xsl:for-each select="$lang-tokens">
+      <xsl:variable name="ltln" select="lower-case(normalize-space(.))"/>
+      <xsl:choose>
+        <xsl:when test="$ltln = $pLang/l">
+          <languageTerm type="code" authority="iso639-2b">
+            <xsl:value-of select="$pLang/l[. = $ltln]/@string"/>
+          </languageTerm>
+        </xsl:when>
+      </xsl:choose>
+    </xsl:for-each>
+  </xsl:template>
+
+  <!-- publisher -->
+  <xsl:template match="dc:publisher">
+    <publisher><xsl:apply-templates/></publisher>
+  </xsl:template>
+
+  <!-- dcterms:created -->
+  <xsl:template match="dcterms:created">
+    <xsl:variable name="date-tokens"
+                  select="if (contains(., ','))
+                          then (tokenize(., ','))
+                          else if (contains(., ';'))
+                            then (tokenize(., ';'))
+                            else (.)"/>
+    <dateCreated encoding="edtf" point="start"><xsl:value-of select="$date-tokens[1]"/></dateCreated>
+    <xsl:if test="count($date-tokens) > 1">
+      <dateCreated encoding="edtf" point="end">
+        <xsl:value-of select="normalize-space($date-tokens[position() = last()])"/>
+      </dateCreated>
+    </xsl:if>
+  </xsl:template>
+  
+  <!-- dcterms:modified -->
+  <xsl:template match="dcterms:modified">
+    <dateModified><xsl:apply-templates/></dateModified>
+  </xsl:template>
+
+  <!-- identifier(s) -->
+  <!-- identifier - location processing -->
+  <xsl:template match="dc:identifier[starts-with(normalize-space(.), 'http://')]">
+    <xsl:variable name="identifier-preview-url" select="replace(., '/cdm/ref', '/utils/getthumbnail')"/>
+    <xsl:variable name="iiif-manifest" select="concat(replace(replace(., 'cdm/ref/collection', 'digital/iiif'), '/id', ''), '/info.json')"/>
+    <url usage="primary" access="object in context"><xsl:apply-templates/></url>
+    <url access="preview"><xsl:value-of select="$identifier-preview-url"/></url>
+    <xsl:if test="normalize-space(.) = $catalog//@id">
+      <url note="iiif-manifest"><xsl:value-of select="$iiif-manifest"/></url>
+    </xsl:if>
+  </xsl:template>
+
+  <xsl:template match="dc:identifier[not(starts-with(normalize-space(.), 'http://'))]">
+    <identifier type="local"><xsl:apply-templates/></identifier>
+  </xsl:template>
+
+  <!-- type(s) starting with capital letters -->
+  <xsl:template match="dc:type[matches(., '^[A-Z]')]">
+    <xsl:variable name="type-tokens" select="tokenize(., ';')"/>
```

Comment:
The sample data gives us an error here: `newspapers` is not a valid `<typeOfResource/>`; e.g.. 
```text

Value 'newspapers' is not facet-valid with respect to enumeration '[text, cartographic, notated music, sound recording-musical, sound recording-nonmusical, sound recording, still image, moving image, three dimensional object, software, multimedia, mixed material, ]'. It must be a value from the enumeration.
```

Reply:
These just need to be /physicalDescription/forms